client/my-sites/activity/activity-log-banner/intro-banner.jsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import CardHeading from 'components/card-heading';
+import DismissibleCard from 'blocks/dismissible-card';
+import ExternalLink from 'components/external-link';
+import {
+	FEATURE_JETPACK_ESSENTIAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	PLAN_JETPACK_PERSONAL_MONTHLY,
+	PLAN_PERSONAL,
+} from 'lib/plans/constants';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { getSiteSlug, isJetpackSite } from 'state/sites/selectors';
+import { isFreePlan } from 'lib/plans';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+class IntroBanner extends Component {
+	recordLearnMore = () =>
+		this.props.recordTracksEvent( 'calypso_activitylog_intro_banner_learn_more' );
+
+	recordUpgrade = () => this.props.recordTracksEvent( 'calypso_activitylog_intro_banner_upgrade' );
+
+	recordDismiss = () => this.props.recordTracksEvent( 'calypso_activitylog_intro_banner_dismiss' );
+
+	renderCardContent() {
+		const { siteIsJetpack, siteIsOnFreePlan, siteSlug, translate } = this.props;
+		const upgradePlan = siteIsJetpack ? PLAN_JETPACK_PERSONAL_MONTHLY : PLAN_PERSONAL;
+		const upgradeFeature = siteIsJetpack
+			? FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY
+			: FEATURE_JETPACK_ESSENTIAL;
+
+		return (
+			<Fragment>
+				<p>
+					{ translate(
+						'The Activity tracks the events that occur on your site so that you don’t have to.'
+					) }
+				</p>
+				<p>
+					{ siteIsOnFreePlan
+						? translate(
+								'With your free plan, you can monitor the 20 most recent ' +
+									'events. A paid plan unlocks more powerful features. ' +
+									'You can access all site activity for the last 30 days ' +
+									'and filter events by type and time range to quickly find ' +
+									'the information you need. '
+						  )
+						: translate(
+								'Explore the list below or filter events by type and ' +
+									'time range to quickly find the information you need. '
+						  ) }
+					<ExternalLink
+						href="https://en.blog.wordpress.com/2018/10/30/introducing-activity/"
+						icon
+						onClick={ this.recordLearnMore }
+						target="_blank"
+					>
+						{ translate( 'Learn more' ) }
+					</ExternalLink>
+					.
```

Comment:
Good call 👍 

Actually, we don't need it anymore since we introduced the external link icon. Removed in https://github.com/Automattic/wp-calypso/pull/28430/commits/46ed8607a948a105a98499cb194738542e3e9e74